### PR TITLE
Test hbg data

### PR DESCRIPTION
**Pull request by sgratiy**

### CODE REVIEW

File: ipfx/dataset/ephys_nwb_data.py [hunk 1]
```diff
@@ -61,12 +61,21 @@ def __init__(self,
...
-        self.nwb = NWBHDF5IO(nwb_file, mode='r').read()
+        
+        if isinstance(nwb_file, str):
+            if load_into_memory:
+                with open(nwb_file, 'rb') as fh:
+                    data = BytesIO(fh.read())
+                _h5_file = h5py.File(data, "r")
+                self.nwb = NWBHDF5IO(path=_h5_file.filename, mode="r+",file=_h5_file).read()
+            else:
+                self.nwb = NWBHDF5IO(nwb_file, mode='r').read()
```
Comment (sgratiy) [inline ipfx/dataset/ephys_nwb_data.py]: why here the mode is 'r' and in other cases 'r+'?

File: ipfx/dataset/ephys_nwb_data.py [hunk 2]
```diff
@@ -61,12 +61,21 @@ def __init__(self,
...
+        elif isinstance(nwb_file, BytesIO):
+            _h5_file = h5py.File(nwb_file, "r")
+            self.nwb = NWBHDF5IO(path=_h5_file.filename, mode="r+",file=_h5_file).read()
+        else:
+            print("Invalid input NWB file, please input either NWB file path or hdf5 obj!")
```
Comment (sgratiy) [inline ipfx/dataset/ephys_nwb_data.py]: maybe 
`raise TypeError(<message>)` 
would be better

Comment (sgratiy) [reply ipfx/dataset/ephys_nwb_data.py]: overall, this entire block of code could be a function that returns nwb object

File: ipfx/dataset/ephys_nwb_data.py (hunk 1 above)
Comment (gnayuy) [reply ipfx/dataset/ephys_nwb_data.py]: change all to "r"

File: ipfx/dataset/ephys_nwb_data.py (hunk 2 above)
Comment (sgratiy) [reply ipfx/dataset/ephys_nwb_data.py]: in any case, looks good. I am fine with merging. Could you please merge it tonight?

Comment (gnayuy) [reply ipfx/dataset/ephys_nwb_data.py]: adjust accordingly

**Pull request merged**